src/ripple/net/impl/RPCCall.cpp

```diff
@@ -309,8 +309,16 @@ class RPCParser
         }
         else
         {
-            std::int64_t uLedgerMin = jvParams[1u].asInt();
-            std::int64_t uLedgerMax = jvParams[2u].asInt();
+            size_t idx = 1;
+            if (jvParams[idx].asString() == jss::marker)
+            {
+                Json::Value marker;
+                marker[jss::ledger] = jvParams[++idx].asInt();
+                marker[jss::seq] = jvParams[++idx].asInt();
+                jvRequest[jss::marker] = marker;
+            }
+            std::int64_t uLedgerMin = jvParams[idx].asInt();
```

Comment:
It took me some time to figure out why this code was even changed. This adds support for markers to the command line syntax of `account_tx`; I was just trying to test things. Reporting mode doesn't need this change in any way, and this would also require a documentation change. I am inclined to switch this snippet of code back to how it was before, because reporting doesn't need it, and the PR is big enough as is. If we want to add support for markers to the command line syntax for `account_tx`, we can do it in another PR.

Reply:
I reverted this change